src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -412,17 +436,24 @@ public void StopReceiving()
             InputFileStream certificate = default,
             int maxConnections = default,
             IEnumerable<UpdateType> allowedUpdates = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            string ipAddress = default,
+            bool dropPendingUpdates = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.